lib/translations/app_localizations.dart

```diff
@@ -0,0 +1,291 @@
+    'bambooDeployment.details': {
+      'en': 'Details',
+    },
+    'bambooDeployment.deploymentState': {
+      'en': 'Deployment state: ',
+    },
+    'bambooDeployment.lifecycleState': {
+      'en': 'Lifecycle state: ',
+    },
+    'bambooPlan.details': {
+      'en': 'Details',
+    },
+    'bambooPlan.state': {
+      'en': 'State: ',
+    },
+    'jenkinsJob.details': {
+      'en': 'Details',
+    },
+    'jiraBuckets.bucket': {
+      'en': 'Bucket',
+    },
+    'jiraBuckets.issues': {
+      'en': 'Issues',
+    },
+    'jiraBuckets.noBuckets': {
+      'en': 'No buckets available',
+    },
+    'sonarQube.details': {
+      'en': 'Details',
+    },
+    'widgetStatus.errorConfiguration': {
+      'en': 'Configuration Error',
+    },
+    'worldClock.loading': {
+      'en': 'Loading...',
+    },
+    'dissmisibleWidgetList.removed': {
+      'en': 'Removed ',
+    },
+    'dissmisibleWidgetList.undo': {
+      'en': 'UNDO',
+    },
+    'serviceCheck.response': {
+      'en': 'Response',
+    },
+    'urlLauncher.toast': {
+      'en': 'URL error occurred',
+    },
+    'iframeEmbed.blankUrl': {
+      'en': 'URL is blank',
+    },
+    'zabbix.history': {
+      'en': 'HISTORICAL DATA',
+    },
+    'zabbix.date': {
+      'en': 'Date',
+    },
+    'widget.notUpdated': {
+      'en': 'Not updated yet',
+    },
+    'settingsScreen.projects': {
+      'en': 'Projects',
+    },
+    'settingsScreen.general': {
+      'en': 'General',
+    },
+    'settingsProjectsScreen.title': {
+      'en': 'Projects',
+    },
+    'settingsProjectsScreen.alert.title': {
+      'en': 'Are you sure?',
+    },
+    'settingsProjectsScreen.alert.confirm': {
+      'en': 'YES',
+    },
+    'settingsProjectsScreen.alert.decline': {
+      'en': 'NO',
+    },
+    'settingsGeneralScreen.title': {
+      'en': 'General',
+    },
+    'settingsGeneralScreen.widgetSorting': {
+      'en': 'Widgets Sorting',
+    },
+    'settingsGeneralScreen.notifications': {
+      'en': 'Notifications',
+    },
+    'settingsGeneralScreen.notificationFrequency': {
+      'en': 'Notification Frequency',
+    },
+    'settingsGeneralScreen.hints': {
+      'en': 'Hints',
+    },
+    'settingsGeneralScreen.alert.confirm': {
+      'en': 'OK',
+    },
+    'settingsSortByWidget.sortBy': {
+      'en': 'Sort By',
+    },
+    'settingsSortByWidget.none': {
+      'en': 'None',
+    },
+    'settingsSortByWidget.name': {
+      'en': 'Name',
+    },
+    'settingsSortByWidget.status': {
+      'en': 'Status',
+    },
+    'settingsSortByWidget.order': {
+      'en': 'Order',
+    },
+    'settingsSortByWidget.asc': {
+      'en': 'Ascending',
+    },
+    'settingsSortByWidget.desc': {
+      'en': 'Descending',
+    },
+    'addConnectionScreen.title.edit': {
+      'en': 'Edit Connection',
+    },
+    'addConnectionScreen.save': {
+      'en': 'SAVE',
+    },
+  };
+
+  String getTranslation(String key) {
+    return _localizedValues[key][locale.languageCode];
```

Comment:
it sould be: `_localizedValues[locale.languageCode][key];`

Reply:
done